### ISSUE TITLE
Please release cp312 wheel

### ISSUE THREAD

**Issue by ThiefMaster** [oracle/python-oracledb#237]
Considering that Python 3.12 is out and you already declare support for it, it would be nice to also have binary wheels for 3.12 available.

This would be especially useful because building it from source takes quite a while on a pretty fast machine:

```
[adrian@eluvian:/tmp]> fc -lEDL -1
25039  5.10.2023 13:12  2:42  pip install oracledb
```

nearly 3 minutes

**Comment by anthony-tuininga** [comment]
We certainly plan to include Python 3.12 in the next release. I'll look into creating wheels for version 1.4.1 as well and get back to you on that.

**Comment by ThiefMaster** [comment]
Great, thanks for looking into it!

**Comment by anthony-tuininga** [comment]
python-oracledb 1.4.2 was just released with Python 3.12 binaries.
